mlibrary/lib.umich.edu
# April 2024 package updates

# Overview
Monthly dependency updates to help maintain lib.umich.edu.

These are the April 2024 updates. [JIRA](https://mlit.atlassian.net/jira/software/projects/WEBSITE/boards/27?selectedIssue=WEBSITE-120)

1 major update (fetch-retry) [changelog](https://github.com/jonbern/fetch-retry/releases/tag/6.0.0)
2 minor updates

https://deploy-preview-286--future-wwwlib-previews.netlify.app/

## Testing
List instructions on how to test the pull request. Some examples:

- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- _How to use the feature_.
